### ISSUE TITLE
Missing country flags

### ISSUE THREAD

**Issue by Dorin-David** [Dorin-David/chess.com-database#1]
The player's country flag is missing; currently, the _alt_ 'flag' text is displayed. 

![Screenshot 2023-03-11 101622](https://user-images.githubusercontent.com/70648503/224478868-7d6dfda5-7cc2-4766-adfc-4a1c052c6381.png)

The component responsible for displaying the flag is `CountryFlag`, located at `src\components\UI\CountryFlag.js`. It uses 
[www.countryflags.io](www.countryflags.io) to display the flags. 

**Fix**

- [ ] display country flag
- [ ] improve component's accessibility (the 'flag' text is not descriptive)
